app/src/main/java/com/zulip/android/networking/GravatarAsyncFetchTask.java

```diff
@@ -47,7 +48,7 @@ public static URL sizedURL(Context context, String url, float dpSize) {
         float px = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP,
                 dpSize, r.getDisplayMetrics());
         try {
-            return new URL(url + "&s=" + px);
+            return new URL(UrlHelper.getSenderAvatarUrl(url)  + "&s=" + px);
```

Comment:
I think we could simply use `addHost` method of UrlHelper (see `RecyclerMessageAdapter.setUpGravatar`).